pkg/planner/core/task.go

```diff
@@ -608,6 +608,13 @@ func (p *PhysicalMergeJoin) Attach2Task(tasks ...base.Task) base.Task {
 }
 
 func buildIndexLookUpTask(ctx base.PlanContext, t *CopTask) *RootTask {
+	tp := t.indexPlan
```

Comment:
what does this piece of code mean?

Reply:
The top-level plan of indexPlan might not be IndexScan, such as Selection-IndexScan. Therefore, I use this piece of code to get the IndexScan plan.